openhtf/core/test_record.py

```diff
@@ -48,6 +48,12 @@ class Attachment(collections.namedtuple('Attachment', 'data mimetype')):
   def sha1(self):
     return hashlib.sha1(self.data).hexdigest()
 
+  def _asdict(self):
+    # Don't include the attachment data when converting to dict.
+    return {
```

Comment:
You'll need to update the tests to handle this not including the data field anymore

Reply:
@shaperilio 
Sorry for the slow reply. Before this PR, attachment data was being removed in PhaseState._asdict(). This PR makes the dictionary format of attachments more consistent. The original motivation for removing attachment data is to ensure that a test state can be rapidly serialized by the station API in order to support frontends. The entire test state might be serialized many times per second to keep a frontend up to date.

If you'd like to create a JSON test record that includes attachments you can add use the OutputToJSON output callback in your test with `inline_attachments` set to True.

Reply:
Thanks @Kenadia